soc/renesas/ra/Kconfig

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) 2023 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+# Copyright (c) 2024 Renesas Electronics Corporation
```

Comment:
Why do we add a copyright without changing anything to the file?

Reply:
We removed it. Thank you for your feedback.